apps/fp-frontend-app/src/behandlingsupport/melding/MeldingIndex.tsx

```diff
@@ -41,12 +44,23 @@ export const MeldingIndex = ({
   meldingFormData,
   setMeldingFormData,
   hentOgSettBehandling,
+  toggleVisUtvidetBehandlingSupportIndexKnapp,
 }: Props) => {
   const intl = useIntl();
   const [showSettPaVentModal, setShowSettPaVentModal] = useState(false);
   const [showMessagesModal, setShowMessageModal] = useState(false);
 
   const navigate = useNavigate();
+  const [top, setTop] = useState<number>();
+
+  const scrollReset = useCallback(() => setTop(0), []);
+
+  useEffect(() => {
```

Comment:
Kva blir dette brukt til?

Reply:
Jeg har kopiert fra historikk.tsx for å fikse på horisontal linjer, men ser at jeg ikke har behov for scrollReset. Litt usikker på hva den gjør men jeg tror ikke det trengs i historikk.tsx og i DokumentIndex.tsx. Hva tenker du?